README.md

```diff
@@ -20,6 +20,26 @@ pip install setuptools
 sudo python setup.py install
 ```
 
+## Configuration
+
+The Registration API makes use of two configuration files. The first is native to the Registration API and is described below. The second forms part of the [NMOS Common Library](https://github.com/bbc/nmos-common) and is described in that repository. Note that you will likely have to configure items in both files.
+
+The native Registration API configuration should consist of a JSON object in the file `/etc/ips-regaggregator/config.json`. The following attributes may be set within the object:
+
+*   **priority:** Sets a priority value for this Registration API instance between 0 and 255. A value of 100+ indicates a development rather than production instance. Default: 100.
```

Comment:
```suggestion
*   **priority:** [integer] Sets a priority value for this Registration API instance between 0 and 255. A value of 100+ indicates a development rather than production instance. Default: 100.
```